sparse_strips/vello_hybrid/src/render.rs

```diff
@@ -325,48 +325,19 @@ impl Renderer {
     /// Render to a texture
     pub fn render_to_texture(
```

Comment:
Should we rename this method since it’s now decoupled from textures and tied to the render pass? Maybe just `render`?